src/fetch.ts

```diff
@@ -0,0 +1,6 @@
+import nodeFetch from 'node-fetch';
```

Comment:
I've created an issue to explore this.

Reply:
@SomeHats while doing due diligence on this, I note that: 

- isomorphic-fetch: 
  - was last published 5 years ago
  - has > 4.5M downloads per week
  - 16 open PRs (the newest is over a year old)
  - 61 open issues and the newest is "Is this library still been maintained?" 🤣 
- node-fetch 
  - was last published 22 days ago
  - has >16M downloads per week
  - PR queue is active
  - Issue queue is active

I don't think it's in Airtable.js's best interest to migrate to what appears to be an unmaintained package, so I'd like to close this "won't fix"

Reply:
fwiw, isomorphic-fetch just re-exports node-fetch (in node) or a polyfill/browser-builtins (in the browser). [node-fetch actually references isomorphic-fetch](https://www.npmjs.com/package/node-fetch#motivation) (and another maybe more suitable option) in its docs. i'm not fussed about the specific thing we use here though - the main thing i'm interested in is that for browser users, we don't include all of `node-fetch` in the bundle unnecessarily.

Reply:
Gotcha! That should be satisfied by the `browserify` operation that will omit `node-fetch` since its value is `false` in package.json's `browser` property: 

```
  "browser": {
    "node-fetch": false,
    "abort-controller": false,
    "./lib/package_version": "./lib/package_version_browser"
  },
```

I checked `airtable.browser.js` and verified that `node-fetch` has an empty module entry 👍